docs/docs/related_projects.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: default
+title: Related projects
+nav_order: 8
+---
+
+# Related projects
+
+If you do want that your gateway should be written in another language, here follows some related projects:
+
+* NodeJS:
+  * [grps-dynamic-gateway](https://github.com/konsumer/grpc-dynamic-gateway)
+* TypeScript / Javascript
+  * Not a direct analog, but [rest2grpc](https://www.npmjs.com/package/rest2grpc) is similar.
+* C++
+  * Envoy proxy [gRPC-JSON transcoder](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/grpc_json_transcoder_filter)
+* Cloud
+  * Google Cloud Platform [HTTP/JSON gRPC transcoding](https://cloud.google.com/endpoints/docs/grpc/transcoding)
```

Comment:
Lets make this less about what language it's written in
```suggestion
# Related projects
- [grps-dynamic-gateway](https://github.com/konsumer/grpc-dynamic-gateway)
   A dynamically configured alternative to the grpc-gateway written in Node.
- [rest2grpc](https://www.npmjs.com/package/rest2grpc)
   A statically configured alternative to the grpc-gateway written in Node.
- The Envoy proxy [gRPC-JSON transcoder](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/grpc_json_transcoder_filter)
   An Envoy proxy filter that translates incoming JSON requests to gRPC and back.
- Google Cloud Platform [HTTP/JSON gRPC transcoding](https://cloud.google.com/endpoints/docs/grpc/transcoding)
   A GCP product that behaves like the grpc-gateway.
```

Feel free to correct the description of your own project, not sure I understand what it does.